src/pages/formFields.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import {
+  TextField,
+  Select,
+  MenuItem,
+  FormControl,
+  InputLabel,
+  Checkbox,
+  FormControlLabel,
+} from '@material-ui/core';
+import 'date-fns';
+import DateFnsUtils from '@date-io/date-fns';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+
+function TextFieldComp(properties) {
+  const { id, label, type } = properties;
+  return (
+    <TextField
+      key={id}
+      id={id}
+      label={label}
+      type={type}
+      InputLabelProps={{ shrink: true }}
+      fullWidth
+      InputProps={{ inputProps: { min: 0, step: 50 } }}
```

Comment:
the `min: 0` is so that the value amount of scholarships can't be below 0. The `step:50` is so that when a user clicks on the up/down arrow for changing values, it increases/decreases by 50

Reply:
That should only be relevant for amount fields though. I would leave that for the Amount text field only. Or better yet, a separate `ScholarshipAmountTextField` function component.